SWLOR.Game.Server/Feature/ChatCommandDefinition/CharacterChatCommand.cs

```diff
@@ -446,5 +450,42 @@ private void Follow()
                     });
                 });
         }
+
+        private void OrderCompanion()
+        {
+            _builder.Create("ordercompanion", "order", "oc")
+                .Description("Orders your active companion to move to perform an interaction with a target. \n" +
+                "Use this command on your henchman to clear its enmity table and cancel all actions.")
+                .Permissions(AuthorizationLevel.All)
+                .RequiresTarget()
+                .Action((user, target, location, args) =>
+                {
+                    var associate = GetHenchman(user);
+                    if (target == associate)
+                    {
+                        Enmity.ClearEnmityTables(associate);
```

Comment:
This is abusable. We shouldn't be letting players clear enmity tables of their droids/beasts.